### DISCUSSION TITLE
Email login and signing transactions

### DISCUSSION THREAD

**Discussion by princessdev0** [Web3Auth/Web3Auth#442] [Help]
Hello can we use web3auth for connecting it to the Dapp using email login and then use the web3auth wallet address to interact with like signing transactions like buying and selling NFT and redeeming coins

**Comment by shahbaz17** [comment]
Hey @TurtleGirlz 

Yes, you can. Please see our demo app https://demo-app.web3auth.io to use Email passwordless and look at the SDK reference https://web3auth.io/docs/sdk/web/core/ to see how it can be done with Email Password as well.
